src/NLog/Targets/Wrappers/AsyncTargetWrapper.cs

```diff
@@ -379,18 +379,18 @@ private void ProcessPendingEvents(object state)
             }
             finally
             {
-                if (TimeToSleepBetweenBatches <= 0 && checkedQueueWasFull)
-                    this.StartInstantWriterTimer(); // Shedule while holding lock, so we don't pile up
+                if (TimeToSleepBetweenBatches <= 0 && wroteFullBatchSize.HasValue && wroteFullBatchSize.Value)
```

Comment:
just a style question. I prefer in this case `wroteFullBatchSize == true`.  Because its shorter and more clear. What do you think?